src/app/settings/change-password.component.ts

```diff
@@ -192,4 +196,15 @@ export class ChangePasswordComponent extends BaseChangePasswordComponent {
             await this.apiService.putEmergencyAccess(details.id, updateRequest);
         }
     }
+
+    private async updateAllResetPasswordKeys() {
+        const orgs = await this.userService.getAllOrganizations();
+
+        for (const org of orgs) {
+            const request = new OrganizationUserResetPasswordEnrollmentRequest();
+            request.resetPasswordKey = null;
+
+            await this.apiService.putOrganizationUserResetPasswordEnrollment(org.id, org.userId, request);
+        }
+    }
```

Comment:
This should be done on the server as part of the rotate operation, if possible. Having multiple actions like this on the client is more fail-prone.

Finally, should we consider updating the reset key for orgs rather than removing it?

Reply:
The reason I put this here is because I believe our end goal will be to update the key rather than removing it (which will require client-side encryption). Phase 1 our of MVP is nulling the keys.

Reply:
I'd argue that we should just go ahead and update it. It's not really any more effort here from what I can tell.

Reply:
Fine by me! Will update task/docs accordingly.

Reply:
@kspearrin - Updated. Please double check my encryption process. I want to make sure I used the correct `encKey` (gets a little confusing)

Removed warning string as we'll be taking care of each re-enrollment automagically.